src/OpenTelemetry/Metrics/Metric.cs

```diff
@@ -126,7 +128,7 @@ public sealed class Metric
                 throw new NotSupportedException($"Unsupported Instrument Type: {instrumentIdentity.InstrumentType.FullName}");
             }
 
-            this.aggStore = new AggregatorStore(instrumentIdentity.InstrumentName, aggType, temporality, maxMetricPointsPerMetricStream, histogramBounds ?? DefaultHistogramBounds, tagKeysInteresting);
+            this.aggStore = new AggregatorStore(instrumentIdentity.InstrumentName, aggType, temporality, maxMetricPointsPerMetricStream, histogramBounds ?? DefaultHistogramBounds, DefaultExponentialHistogramMaxBuckets, tagKeysInteresting);
```

Comment:
Just leaving a note: We would have to set the AggregationType to ExponentialHistogram in the ctor.

You might have planned this for another PR. 🙂

Reply:
Yea good call. Hadn't actually thought about this, but I'll grab it in another PR. I'll be adding tests as I go. This will certainly be caught.